__tests__/lint/assertions-vendor-extensions/.redocly.yaml

```diff
@@ -0,0 +1,36 @@
+apis:
+  main:
+    root: ./openapi.yaml
+
+rules:
+  assert/vendor-extension:
+    subject:
+      type: VendorExtension
+      property: prop
+      filterInParentKeys: ['x-my-extension']
+    severity: error
+    assertions:
+      enum: [foo, bar]
+  assert/vendor-extension-in-info-section:
+    subject:
+      type: VendorExtension
+      property: prop2
+      filterInParentKeys: ['x-my-extension']
+    where:
+      - subject:
+          type: Info
+        assertions:
+          defined: true
+    severity: error
+    assertions:
+      enum: [foo, bar]
+
+  # built-in vendor extension should be processed as generic VendorExtension too
```

Comment:
Do we want to keep this?

Reply:
Yes, I still want to get it working hopefully.

Reply:
Didn't get it working. I'll merge with this commented out for now. Hopefully we can fix it later.